amr-wind/wind_energy/ABLBoundaryPlane.cpp

```diff
@@ -884,18 +879,6 @@ void ABLBoundaryPlane::populate_data(
             continue;
         }
 
-        // Ensure the fine level does not touch the inflow boundary
-        if (lev > 0) {
-            const amrex::Box& minBox = m_mesh.boxArray(lev).minimalBox();
```

Comment:
are you sure we can remove this continue?

Reply:
ok I see the problem. If the inflow boundary plane data doesn't have a fine level, then there's a problem. Great catch!

Reply:
I put the continue back in